### DISCUSSION TITLE
Why variant order is ignored?

### DISCUSSION THREAD

**Discussion by chojnicki** [tailwindlabs/tailwindcss#8756] [Help]
I found similar questions but this does not help me in my situation https://github.com/tailwindlabs/tailwindcss/discussions/4857 https://github.com/tailwindlabs/tailwindcss/discussions/5350

https://play.tailwindcss.com/8GRd9lc7ao

I want to `focus-within` have same priority as `focus`, so higher than hover. Currently on hover, focus-within is ignored... I changed variantOrder but nothing changes. This is CSS result:

```
.focus-within\:border-green-500:focus-within {
  --tw-border-opacity: 1;
  border-color: rgb(34 197 94 / var(--tw-border-opacity));
}

.hover\:border-blue-500:hover {
  --tw-border-opacity: 1;
  border-color: rgb(59 130 246 / var(--tw-border-opacity));
}

.focus\:border-green-500:focus {
  --tw-border-opacity: 1;
  border-color: rgb(34 197 94 / var(--tw-border-opacity));
}

```

Only fix I can use is stacking variants, so `hover:focus-within:border-green-500` but it's duplicating color and extra code that should not be necessary. Is there a way to force JIT to treat `focus-within` same way as `focus`?

**Comment by crswll** [comment]
I'm not sure this is configurable since v3. I tried to do the same the other day with no luck then asked about it and basically suggested at what you arrived at. Here's a link to the quick discussion if it's useful for you: https://discord.com/channels/486935104384532500/546706299010678784/986067521205592065
